app/controllers/admin/field_types_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+module Admin
+  class FieldTypesController < ApplicationController
+    before_action :load_field, only: %i(show)
+
+    def index; end
+
+    def new
+      @field_type = FieldType.new
+    end
+
+    def show
+      @pagy, @field_types = pagy @field.field_types, items: Settings.PERPAGE_8
+    end
+
+    def create
+      @field = Field.find_by(id: params[:field_id])
+      @field_type = @field.field_types.build(field_type_params)
+      @field_type.is_availible ||= true
+      build_prices(params)
+      if @field_type.save
+        flash[:success] = t("admin.field_type.create_success")
+        redirect_to admin_field_type_path(params[:field_id])
+
+      else
+        flash[:danger] = t("admin.field_type.create_fail")
```

Comment:
default, render :new will return status code is 200, this don't inform any errors, add status: :... to show errors